lib/core/list-files-and-perform-sanity-checks.ts

```diff
@@ -0,0 +1,33 @@
+import glob from "glob";
```

Comment:
It's a pretty straightforward file, but wondering if we should add any test coverage for the 0 and 1 cases?